QuizWhois/QuizWhois/Properties/launchSettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:20653",
+      "sslPort": 44330
+    }
+  },
+  "profiles": {
+    "IIS Express": {
```

Comment:
Do we really need the IIS section?

Reply:
I believe IIS section is auto-generated

Reply:
... and it is not a crime to remove it to avoid confusing